src/serlio/utils/Utilities.cpp

```diff
@@ -61,62 +62,17 @@ std::wstring getPluginRoot() {
 		throw std::runtime_error("failed to get plugin location");
 	}
 
-	_splitpath_s(dllPath, drive, 8, dir, _MAX_PATH, 0, 0, 0, 0);
-	std::wstring rootPath = prtu::toUTF16FromOSNarrow(drive);
-	rootPath.append(prtu::toUTF16FromOSNarrow(dir));
+	rootPath = std::filesystem::path(dllPath).parent_path();
```

Comment:
please remove the unused variables `drive` and `dir`

Reply:
removed unused variables